prow/config.yaml

```diff
@@ -26,6 +26,66 @@ presubmits:
   #   spec:          If this exists then run a kubernetes pod with this spec.
   #                  Otherwise, run a Jenkins job.
 
+  istio/api:
+  - name: api-presubmit
+    agent: kubernetes
+    context: prow/api-presubmit.sh
+    always_run: true
+    rerun_command: "/test api-presubmit"
+    trigger: "(?m)^/(retest|test (all|api-presubmit))?,?(\\s+|$)"
+    branches:
+    - master
+    - release-0.2
+    spec:
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.2.1
+        args:
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--clean"
+        - "--timeout=20"
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/service-account/service-account.json
+        volumeMounts:
+        - name: service
+          mountPath: /etc/service-account
+          readOnly: true
+        - name: api-e2e-rbac-kubeconfig
```

Comment:
Remove this part too if we don't need a cluster and code coverage for now

Reply:
Done